packages/x-components/src/plugins/__tests__/x-plugin-mixin.spec.ts

```diff
@@ -1,32 +1,13 @@
-import { mount, shallowMount, Wrapper } from '@vue/test-utils';
+import { mount, Wrapper } from '@vue/test-utils';
```

Comment:
Can we move the render function outside of the describe and the interfaces too? Like  we have it in the other specs

Reply:
Are you talking about `renderOriginComponent`? I wouldn't, that testing factory only covers tests for the origin, which are encapsulated inside the `describe('testing origin',` block. In other tests we use a single render function for all tests, but this plugin mixin test needs different components for each one.

Reply:
yes, you are right, I didn't think on that. Thank you, you make me see the light